core/src/consensus/tendermint/stake/action_data.rs

```diff
@@ -87,9 +88,7 @@ impl<'a> StakeAccount<'a> {
     }
 }
 
-pub struct Stakeholders {
```

Comment:
I think these changes should be applied on 96af4100f9f7f16962dbb9b4091c3a5cc5f9c9b8